module2/homework/insensitivePalindrom/checkPalindrome.cpp

```diff
@@ -0,0 +1,18 @@
+#include "checkPalindrome.hpp"
+
+#include <algorithm>
+#include <iostream>
+#include <string>
```

Comment:
String is redundant here since it's in the header.

Reply:
+1